src/routes/+page.svelte

```diff
@@ -38,6 +38,8 @@
         slot="image"
         xmlns="http://www.w3.org/2000/svg"
         viewBox="0 0 24 24"
+        role="img"
+        aria-label="disc image"
```

Comment:
I think you need to add a title tag to the svg 🤔 [[1](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title?retiredLocale=de)]